database/migration/migrateData.ts

```diff
@@ -854,6 +856,83 @@ const migrateData = async () => {
     }
   }
 
+  // v0.8.0
+  if (databaseVersionLessThan('0.8.0')) {
+    console.log('Migrating to v0.8.0...')
+
+    console.log(' - Creating permission_flattened view')
+    await DB.changeSchema(`
+        CREATE INDEX IF NOT EXISTS "i_review_status_history_status" ON review_status_history (status);
+        CREATE OR REPLACE view permission_flattened as
+        SELECT "user".id as user_id, permission_join.organisation_id as organisation_id, permission_policy.id as permission_policy_id, template_permission.template_id as template_id  
+        FROM "user" 
+        JOIN permission_join ON permission_join.user_id = "user".id
+        JOIN permission_name ON permission_name.id = permission_join.permission_name_id
+        JOIN permission_policy ON permission_policy.id = permission_name.permission_policy_id
+        JOIN template_permission ON template_permission.permission_name_id = permission_name.id
+      `)
+
+    console.log(
+      ' - Updating policies to replace jwtPermission_array_bigint_template_ids with query rathern then in statement matching arrays in JWT'
+    )
+    const policies = (
+      await DBConnect.query({
+        text: `SELECT id, rules FROM permission_policy`,
+      })
+    ).rows as { id: number; rules: object }[]
+
+    // This is added as an optimisation, rather then doing "in" clause with array values from JWT
```

Comment:
I was wondering if I need to add this $in replacement to some 'core' place for policies, but I think they should get updated with migration ? Also any future policies should use this